.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/victoriaklingenstein/.local/share/virtualenvs/CoTrip-TXuPc5lv/bin/python"
+}
```

Comment:
Can you delete these settings files?

Reply:
Deleted. Not sure where these came from.